UPGRADE-15.0.md

```diff
@@ -832,6 +832,13 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
     -   `getAllOfferedProducts()` method was removed, use `getAllOfferedProductsPaginated()` instead
 -   see #project-base-diff to update your project
 
+#### images are no longer processed by PHP to avoid quality decrease ([#3169](https://github.com/shopsys/shopsys/pull/3169))
```

Comment:
Shouldn't it be in the form of `do something`?  (like stop processing images by PHP to avoid decreasing quality)

Reply:
I always forget about this :smile: yes, you are right, fixed, thanks! 10e41fa816